components/slot-fill/README.md

```diff
@@ -47,3 +47,8 @@ You can either use the Fill component directly, or a wrapper component type as i
 The `SlotFillProvider` component does not accept any props.
 
 Both `Slot` and `Fill` accept a `name` string prop, where a `Slot` with a given `name` will render the `children` of any associated `Fill`s.
+
+`Slot` also accepts a `bubblesVirtually` event which changes the event bubbling behaviour:
+
+ - By default, events will bubble to their parents on the DOM hierarchy (native event bubbling)
+ - if `bubblesVirtually` is set to true. Events will bubble to their virtual parent in the React elements hierarchy instead.
```

Comment:
Minor grammar: Capitalize "If", change dot (".") after "true" to comma (","), lowercase "events" to keep within the same sentence.